superset-frontend/src/SqlLab/components/App/index.jsx

```diff
@@ -125,6 +127,7 @@ class App extends React.PureComponent {
     ) {
       this.showLocalStorageUsageWarning(
         this.props.localStorageUsageInKilobytes,
+        this.props.queries?.lenghth || 0,
```

Comment:
Whoa, interesting catch here. 

CC @justinpark in case this poses any risk... or actually fixes something, since this line was ALWAYS returning 0 in its current implementation.